Gemfile

```diff
@@ -26,6 +26,8 @@ gem "stimulus-rails"
 
 # Build JSON APIs with ease [https://github.com/rails/jbuilder]
 gem "jbuilder"
+gem 'rubocop'
```

Comment:
Do we need both these? I have a feeling that rubocop is included in rubocop-rails.